src/validators/rsavalidator.cpp

```diff
@@ -44,12 +44,12 @@ RSAValidator::~RSAValidator() {
 
 bool RSAValidator::Verify(json_t *jsonHeader, const uint8_t *header, size_t num_header,
                           const uint8_t *signature, size_t num_signature) {
-  EVP_MD_CTX evp_md_ctx;
-  EVP_MD_CTX_init(&evp_md_ctx);
-  EVP_VerifyInit_ex(&evp_md_ctx, md_, NULL);
-  bool valid = EVP_VerifyUpdate(&evp_md_ctx, header, num_header) == 1 &&
-    EVP_VerifyFinal(&evp_md_ctx, signature, num_signature, public_key_) == 1;
-  EVP_MD_CTX_cleanup(&evp_md_ctx);
+  EVP_MD_CTX *evp_md_ctx = EVP_MD_CTX_new();
+  EVP_MD_CTX_init(evp_md_ctx);
```

Comment:
This isn't actually needed now because EVP_MD_CTX_new() already initializes everything the way it should be.